### PR TITLE
test_collector change in FTP code for TERRAMA2_DATA_DIR

### PR DESCRIPTION
**Pull request by BiancaMaciel**
- Compilation correction for python library and Code of Nova Friburgo and fire risk examples

### CODE REVIEW

File: src/unittest/collector/IntRasterTs.cpp [hunk 1]
```diff
@@ -58,7 +58,8 @@ void addInput(std::shared_ptr<terrama2::services::collector::core::DataManager>
   terrama2::core::DataProviderPtr dataProviderPtr(dataProvider);
   dataProvider->id = 2;
   dataProvider->name = "DSA curso";
-  dataProvider->uri = "ftp://ftp:JenkinsD%40t%40@jenkins-ftp.dpi.inpe.br:21/terrama2/";
+  //dataProvider->uri = "ftp://ftp:JenkinsD%40t%40@jenkins-ftp.dpi.inpe.br:21/terrama2/";
+  dataProvider->uri = "file://"+TERRAMA2_DATA_DIR+"/";
   dataProvider->intent = terrama2::core::DataProviderIntent::COLLECTOR_INTENT;
   dataProvider->dataProviderType = "FTP";
```
Comment (janosimas) [inline src/unittest/collector/IntRasterTs.cpp]: @BiancaMaciel  you need to alter the DataProviderType too

**Comment by BiancaMaciel** [comment]
Jenkins rebuild

**Comment by BiancaMaciel** [comment]
jenkins rebuild

**Comment by BiancaMaciel** [comment]
jenkins rebuild

**Comment by BiancaMaciel** [comment]
jenkins rebuild

**Pull request merged**